django/db/backends/oracle/introspection.py

```diff
@@ -22,15 +23,11 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
         cx_Oracle.TIMESTAMP: 'DateTimeField',
     }
 
-    try:
+    with suppress(AttributeError):
         data_types_reverse[cx_Oracle.NATIVE_FLOAT] = 'FloatField'
```

Comment:
Let's revert this file and have an Oracle expert handle the changes in https://code.djangoproject.com/ticket/27822#ticket.